### PR TITLE
Nodejs ubuntu

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by aaron-prindle** [comment]
This PR builds on top of https://github.com/GoogleCloudPlatform/debian-docker/pull/105
It adds //reproducible:nodejs_ubuntu and //dockerfile_build:ubuntu_build rules

File: reproducible/mknodeimage.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#     usage
+# fi
+
+WORKDIR="/workspace/nodejs_build"
+OUTPUTDIR="$WORKDIR/nodejs"
+mkdir -p "$OUTPUTDIR"
+cd "$WORKDIR"
+
+# TODO(aaron-prindle) make sure the rootfs is structured properly (isn't rn)
+curl -o node-v6.11.4.tar.gz https://nodejs.org/dist/v6.11.4/node-v6.11.4.tar.gz
```
Comment (dlorenc) [inline reproducible/mknodeimage.sh]: Could we grab this via WORKSPACE instead?

Comment (aaron-prindle) [reply reproducible/mknodeimage.sh]: done

**Comment by aaron-prindle** [comment]
added rules for all 4 required images:
//reproducible:ubuntu
//dockerfile_build:ubuntu_build
//reproducible:nodejs_ubuntu
//reproducible:nodejs_ubuntu_build

**Pull request closed without merging**